### PR TITLE
Created SortCharsByFrequencyController.

### PR DESCRIPTION
**Pull request by xedisu**
Created SortCharsByFrequencyInputPage.html.
Created SortCharsByFrequencyOutputPage.html
Created StringDTO.java

### CODE REVIEW

File: src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @Autowired
+    SortCharsByFrequencyService sortCharsByFrequencyService;
+
+    @GetMapping("/SortCharsByFrequency")
+    public String getSortCharsByFrequency(Model model) {
+        StringDTO stringDTO = new StringDTO();
+
+        model.addAttribute("stringDTO", stringDTO);
+
+        return "SortCharsByFrequencyInputPage";
```
Comment (BogdanTanase) [inline src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java]: Does this need to be with capital letter?

File: src/main/java/com/algorithmtron/model/StringDTO.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class StringDTO {
```
Comment (BogdanTanase) [inline src/main/java/com/algorithmtron/model/StringDTO.java]: Consider naming this DTO in a more appropriate matter to better reflect where it is used.

**Review by BogdanTanase — CHANGES_REQUESTED** [review]

File: src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java (hunk 1 above)
Comment (xedisu) [reply src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java]: you're right! it slipped my eyes. will change. :)

File: src/main/java/com/algorithmtron/model/StringDTO.java (hunk 2 above)
Comment (xedisu) [reply src/main/java/com/algorithmtron/model/StringDTO.java]: Okay, i'll try to find a more suggestive name for it!

**Review by BogdanTanase — APPROVED** [review]

**Pull request merged**